src/main/java/games/strategy/triplea/ui/menubar/ExportMenu.java

```diff
@@ -308,13 +309,12 @@ private void createAndSaveStats(final boolean showPhaseStats) {
       text.append("\n");
       clone.getHistory().gotoNode(clone.getHistory().getLastNode());
       @SuppressWarnings("unchecked")
-      final Enumeration<HistoryNode> nodes =
-          ((DefaultMutableTreeNode) clone.getHistory().getRoot()).preorderEnumeration();
+      final Enumeration<TreeNode> nodes = ((DefaultMutableTreeNode) clone.getHistory().getRoot()).preorderEnumeration();
```

Comment:
I don't think this is neccessary. Casts are easy to understand, and we are iterating over this enumeration anyway.